tests/integration/devfile/cmd_devfile_create_test.go

```diff
@@ -34,6 +34,17 @@ var _ = Describe("odo devfile create command tests", func() {
 		helper.CommonAfterEach(commonVar)
 	})
 
+	Context("when .gitignore file exists", func() {
+		It("checks that .odo/env exists in gitignore", func() {
+			helper.Cmd("odo", "create", "nodejs", "--project", commonVar.Project, cmpName).ShouldPass()
```

Comment:
In one of the discussions, you mentioned that we should try to use devfile available in the examples. Should we try to do that here instead of pulling a devfile from the registry?

Reply:
yes, I also mentioned that I'm not going to do this in this PR. That I would do it separately.
Pr with git changes like this can't be open for too long. It is already impossible to rebase this when someone adds something to this file in a different PR.